setupext.py

```diff
@@ -205,7 +204,7 @@ def check_for_pyembree(std_libs):
     embree_libs = []
     embree_aliases = {}
     try:
-        _ = resource_filename("pyembree", "rtcore.pxd")
+        _ = importlib.resources.files("pyembree") / "rtcore.pxd"
```

Comment:
this *doesn't* need changes on the pyembree side, right?

Reply:
It doesn't. It works exactly like `pkg_resources` except that it returns a path.
In hindsight it's completely useless (in both implementations) to look for an exact file like we do here, because both implementations are happy to produce a path to a file even if it's not there (though they _do_ error with `ModuleNotFoundError` if `pyembree` isn't installed).